### ISSUE TITLE
Automate Documentation

### ISSUE THREAD

**Issue by xadamxk** [xadamxk/HFX2.0#25]
Find a library to automate wiki documentation.

**Comment by xadamxk** [comment]
Don't think this is needed anymore unless it would be possible to scrap through all the features and get the name, description, and what page they run on. Do you think that would be worth it to autogenerate a markdown page for all the features? @TyrantKingBen

**Comment by TyrantKingBen** [comment]
That could be useful (certainly would spruce up the current README) and should be relatively easy. On a side-note I started adding documentation to the module, but only made it through `HFX.Utils`.

**Comment by xadamxk** [comment]
As far as framework changes go, I think I want to focus on this next.
Do you have any recommendations before I go this route? @TyrantKingBen

**Comment by TyrantKingBen** [comment]
Not really. You need to mock/fake `chrome` (`global.chrome = undefined` suffices), then you can import/require `src/Features` and generate markdown from that (e.g. the `_name`, `_description`, etc.).
